controls/C-0241-hostilemultitenantworkloads.json

```diff
@@ -0,0 +1,20 @@
+{
+    "controlID": "C-0241",
+    "name": "Hostile multi-tenant workloads",
+    "description": "Currently, Kubernetes environments aren't safe for hostile multi-tenant usage. Extra security features, like Pod Security Policies or Kubernetes RBAC for nodes, efficiently block exploits. For true security when running hostile multi-tenant workloads, only trust a hypervisor. The security domain for Kubernetes becomes the entire cluster, not an individual node.\n\n For these types of hostile multi-tenant workloads, you should use physically isolated clusters. For more information on ways to isolate workloads, see Best practices for cluster isolation in AKS.",
+    "long_description": "",
+    "remediation": "",
+    "manual_test": "",
+    "references": [
+        "<https://docs.microsoft.com/en-us/azure/aks/operator-best-practices-cluster-isolation>"
+    ],
+    "attributes": {
+        "armoBuiltin": true
+    },
+    "rulesNames": [
+        "rule-hostile-multitenant-workloads"
+    ],
```

Comment:
Question - Do we need to have a rule here or will it not work if it's empty? I haven't tried it myself but as we have no check we're automating we don't theoretically need a rule. If we need it for technical reasons then fine

Reply:
Maybe we can leave this empty rule, in case we find some action to apply in the near future. WDYT?

Reply:
I tried running it without a rule and it didn't work so I guess we need this empty rule anyways :)